tests/Constraints/CoerciveTest.php

```diff
@@ -128,7 +128,7 @@ public function getValidCoerceTests()
                   "boolean":"true",
                   "object":{},
                   "array":[],
-                  "null":null,
+                  "null":"",
```

Comment:
Added a post-validate test for null.

I also removed what looked like a duplicated block of explicit tests in the same section of code. Not sure if there was any purpose for it...